node/mdSchema.ts

```diff
@@ -167,6 +168,10 @@ export const schemas = [
           type: ['string', 'null'],
           title: 'Phone Number',
         },
+        customFields: {
```

Comment:
We should add this field to the organizations schema too

Reply:
✅ Added those fields